src/views/advanced-search/results.html

```diff
@@ -87,21 +88,22 @@ <h1 class="govuk-heading-l">Advanced company search</h1>
             <input type="button" class="govuk-button govuk-button--secondary" value="Clear search" onclick="clearForm(this.form)"></input>
         </div>
       </form>
-    </div>
-
-    <div class="govuk-grid-two-thirds" style="padding-top: 45px;">
-
-      {% if searchResults.length > 0 %}
-        {{ govukTable ({
-            rows: searchResults,
-            classes: 'govuk-!-font-size-16 govuk-!-width-one-half'
-        }) }}
-      {% else %}
-        <h2 class="govuk-heading-l" style="overflow-wrap: break-word;">No results found</h2>
-        <p class="govuk-body">Please try searching again using a different search criteria.</p>
-      {% endif %}
 
     </div>
-
+    <div class="govuk-grid-two-thirds two-thirds-div" style="padding-top: 45px;">
+      <div>
+        {% if searchResults.length > 0 %}
+          {{ govukTable ({
+              rows: searchResults,
+              classes: 'govuk-!-font-size-16 govuk-!-width-one-half'
+          }) }}   
+          <p class="govuk-body">{{ lastUpdatedMessage }}</p>  
```

Comment:
Can use the global variable set in `app.ts` instead of creating a variable in the controller `{{ ADVANCED_SEARCH_LAST_UPDATED }}`